doc/rst/language/spec/statements.rst

```diff
@@ -798,6 +798,17 @@ This program is equivalent to:
       In M2's main.
       In M1's foo.
 
+The ``module-or-enum-name`` in a ``use`` statement must begin with one of
+the following:
+
+ * a top-level module name
+ * a submodule of the current module
+ * a module name currently in scope due to another ``use`` statement
+ * any number of ``super`` components to indicate a number of parents of
+   the current module (e.g. ``super.super.SomeModule``)
```

Comment:
A potential gap by omission: `super` can appear in places other than the prefix of the expression as well, right?  That is, one could say `use this.super.x;` or `use M.super.N.super.x;`?

Reply:
These are currently errors

Reply:
(result in compilation error)

Reply:
Are they intended to be, though, or is that just a limitation in the current implementation?  (taking the philosophy of "the spec should describe the language as it's intended more than as it's implemented").

Reply:
They are currently intended to be an error. We might change that (and I wouldn't necessarily be opposed to) but we'd have to think about it etc etc.